types/office-js-preview/index.d.ts

```diff
@@ -72879,6 +72644,14 @@ declare namespace Word {
         */
         toJSON(): Word.Interfaces.SettingData;
     }
+    class SettingCustom {
```

Comment:
SettingCustom and Setting are another pair of odd, undescribed entities. See my other comments.